services/app-api/serverless.yml

```diff
@@ -31,9 +31,7 @@ custom:
       - "SizeRestrictions_BODY"
   serverlessTerminationProtection:
     stages:
-      - main
-      - val
-      - production
+      - nonsense
```

Comment:
had to put something there to disable termination protection on all other stages.